tests/Command/AbstractCommandTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Tests\Command;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+abstract class AbstractCommandTest extends KernelTestCase
+{
+    protected function setUp(): void
+    {
+        if ('Windows' === \PHP_OS_FAMILY) {
+            $this->markTestSkipped('`stty` is required to test this command.');
```

Comment:
Moving this to the abstract class is wrong. Not all command tests require `stty`. And AFAICT, testing the `list:users` command will not require stty as it is not an interactive command.

Reply:
Mmmmm, you are right! Anybody willing to make a Pull Request to fix this? Thanks!

Reply:
Fixed in #1230